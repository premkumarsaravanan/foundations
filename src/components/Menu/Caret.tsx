import * as React from 'react'

export interface CaretProps {
  isActive: boolean
}

export const Caret: React.SFC<CaretProps> = ({ isActive }) => (
  <span className="caret-container">
    <span className={`"caret-arrow" ${isActive ? 'caret-active' : ''}`} />
  </span>
)
