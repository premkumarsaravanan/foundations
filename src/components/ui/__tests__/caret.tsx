import * as React from 'react'
import { shallow } from 'enzyme'
import Caret from '../caret'
import toJson from 'enzyme-to-json'

describe('Caret', () => {
  it('should match a snapshot', () => {
    expect(toJson(shallow(<Caret isActive />))).toMatchSnapshot()
  })
})
