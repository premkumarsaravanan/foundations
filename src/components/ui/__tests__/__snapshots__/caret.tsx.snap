// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Caret should match a snapshot 1`] = `
<span>
  <span
    className="undefined undefined"
  />
</span>
`;
